### ISSUE TITLE
Use of PyTorch loss functions inside Keras

### ISSUE THREAD

**Issue by emi-research-dl** [keras-team/keras-core#902]
Hi!! Is possible to use PyTorch loss functions like https://docs.monai.io/en/stable/losses.html inside Keras Core?

Thanks in advanced!
